lib/presentation/widgets/m3_widgets/bottom_navigation_bar/bottom_nav_items.dart

```diff
@@ -7,12 +7,31 @@ enum BottomNavItems {
   trips(asset: AppPaths.luggage),
   expenses(asset: AppPaths.menu),
   community(asset: AppPaths.community),
-  statistic(asset: AppPaths.statistic),
-  recommended(asset: AppPaths.recommended);
+  tripStatistic(asset: AppPaths.statistic),
+  places(asset: AppPaths.places);
 
   const BottomNavItems({required this.asset});
 
   final String asset;
+
+  static const List<BottomNavItems> homePageNavItems = [
+    dashboard,
+    trips,
+  ];
+  static const List<BottomNavItems> tripPanelItems = [
+    community,
+    tripStatistic,
+    places,
+  ];
+
+  static const List<BottomNavItems> allNavItems = [
+    dashboard,
+    trips,
+    expenses,
+    community,
+    tripStatistic,
+    places,
+  ];
```

Comment:
why we need this? If we need all the enum we can just do `BottomNavItems.values`